### ISSUE TITLE
Problems sending AJAX

### ISSUE THREAD

**Issue by Vortelf** [StephanWagner/jBox#73]
Hello,
I have problem sending AJAX request. I don't thing I get it how exactly to send the data.

Here's the button which triggers the JS event.
```
<a class="btn btn-default history" data-url="/Properties/CallHistory/{{$property->id}}" title="История">
```
Here's the script.
```
var HistoryModal =  $('#History-box').jBox('Modal', {
		ajax: {
			getURL: "data-url",
		},
		position:  { x: 'left' , y: 'top' },
		offset: { x: 50, y: 50 },
		draggable: true,
		overlay: false,
		width: 'auto',
		maxWidth: 1000,
		maxHeight: 500,
		responsiveWidth: true,
		responsiveHeight: true,
		closeOnClick: 'body',
		closeButton: 'box',
	});
	$('.history').click(function () {
		HistoryModal.toggle();
	});
```

Any idea how to make this work?

EDIT: Found a workaround. Try the console I found out the Modal.source is undefined which means that the code on line 1528 cannot get the attributes from the source object.
```
	$('.history').click(function () {
                HistoryModal.source = $(this); //This line here
		HistoryModal.toggle();
	});
```

Don't know if I am using it wrong or there is actually an issue.

**Comment by StephanWagner** [comment]
Hi Vortelf,

yeah, you figured it out, the source element was not set. Thats because a source element only gets set when jBox is attached to an element, you could also just attach jBox to the elements and it would  work just fine: https://jsfiddle.net/StephanWagner/f1a1quss/1/

Also, I just noticed that there is a bug when setting draggable: true. This will get fixed in 0.4.5, you can download master brach if you like, I just fixed it there.

The responsive width seems to be bugged too: https://jsfiddle.net/StephanWagner/f1a1quss/2/
Must have something todo with the inside position. I'll mark this issue as a bug and try to fix this soon.

**Comment by Vortelf** [comment]
Dunno, the responsive width is working fine for me.

I'm struggling with the draggable - I tried to attach it to an element from the ajax response, but it din't work so I gave up that idea.

**Comment by StephanWagner** [comment]
thats because it creates the draggable when its being initialised, and it cant find the element as its not in the content yet. I'll fix this in a later release.

**Comment by StephanWagner** [comment]
Should be fixed now in the master branch and will be added to v0.4.5.
draggable now takes the jQuery selector though instead of a jQuery element, e.g. draggable: '.myHandle'

**Comment by StephanWagner** [comment]
Should be fixed now in v0.4.5
